plugins/inventory/digitalocean.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c), Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+name: digitalocean
+plugin_type: inventory
+author:
+  - Janos Gerzson (@grzs)
+short_description: DigitalOcean Inventory Plugin
+version_added: "1.1.0"
+description:
+  - DigitalOcean (DO) inventory plugin.
+  - Acquires droplet list from DO API.
+  - Uses configuration file that ends with C((do_hosts|digitalocean|digital_ocean).(yaml|yml)).
+extends_documentation_fragment:
+  - community.digitalocean.digital_ocean.documentation
+  - constructed
+  - inventory_cache
+options:
+  plugin:
+    description:
+      - The name of the DigitalOcean Inventory Plugin,
+      - this should always be 'community.digitalocean.digitalocean'.
+    required: true
+    choices: ['community.digitalocean.digitalocean']
+  api_token:
+    description:
+     - DigitalOcean OAuth token.
+    required: True
+    type: str
+    aliases: [ oauth_token ]
+    env:
+      - name: DO_API_TOKEN
+  attributes:
+    description: >-
+      Droplet attributes to add as host vars to each inventory host.
+      Check out the DO api docs for full list of attributes at
```

Comment:
```suggestion
      Check out the DO API docs for full list of attributes at
```
I only marked some instances of problems in my first review, you need to check whether they apply to more places.

Reply:
sorry, I was lazy. Now I checked all occurrences.